app/views/organisms/_searchbar.html.erb

```diff
@@ -3,7 +3,7 @@
       <input type="text" name="search_name" >
       <input type="submit" value="Search for Organisms">
   </form>
-  <% if params[:search_name] != "" %>
+  <% if !["", nil].include?(params[:search_name]) %>
```

Comment:
I am mainly trying out this inline-review feature. 😁  isn't if ! better written as unless ?

Reply:
yes you're right, will change it!

Reply:
done